client/src/pages/ProductDetails.js

```diff
@@ -0,0 +1,62 @@
+import Header from '../components/header';
+import { useState, useEffect } from 'react';
+import { readShoesListId } from '../data';
+import { useParams } from 'react-router-dom';
+
+export default function ProductDetails() {
```

Comment:
It looks like this should be ShoeDetails? I'm guessing you'll get there as you add the other products.